### ISSUE TITLE
Unmaintained libraries

### ISSUE THREAD

**Issue by jonathan-s** [shahraizali/awesome-django#9]
Do you accept pull requests for removing stuff that seem unmaintained? What would be a reasonable cutoff?

**Comment by shahraizali** [comment]
Yes, Definitely! we want to keep a list of well maintained and still relevant packages. If you have encountered any that not feel free to create a PR. We can definitely call them out and move to the bottom if not remove completely.
